src/App.js

```diff
@@ -6,15 +6,26 @@ import Navbar from 'react-bootstrap/Navbar'
 import Nav from 'react-bootstrap/Nav'
 import NavDropdown from  'react-bootstrap/NavDropdown'
 import {Component} from "react";
+import { BrowserRouter, Route, Switch} from "react-router-dom";
+import Menu from "./components/Menu";
+import Map from './components/Map';
+import Home from './components/Home';
 
 class App extends Component {
   render() {
-      return (
-        <div className="App">
-            <Header />
-        </div>
-      );
-    }
+    return (
+      <div className="App">
+        <Header />
```

Comment:
Something's up with the Header clipping over the menu and map pages.

Reply:
you and esam mentioned that you had a fix about position fix or something?